.azure/pipelines/Phonebook.preview.yml

```diff
@@ -211,8 +211,6 @@ jobs:
     displayName: 'Delay Deletion of Preview Environment'
     pool: server
     dependsOn: deploy_preview
```

Comment:
Bitte merge Fehler beheben

Reply:
Should be fine now?

Reply:
yep should be fine, although no changes on this file would be even better.